src/parse.rs

```diff
@@ -651,10 +647,11 @@ impl<'a> FirstPass<'a> {
                 b'`' => {
                     self.tree.append_text(begin_text, ix);
                     let count = 1 + scan_ch_repeat(&bytes[ix+1..], b'`');
+                    let preceeded_by_backslash = ix > 0 && bytes[ix - 1] == b'\\';
                     self.tree.append(Item {
                         start: ix,
                         end: ix + count,
-                        body: ItemBody::MaybeCode(count),
+                        body: ItemBody::MaybeCode(count, preceeded_by_backslash),
```

Comment:
Hmmmm, this is probably not correct. `` \\`foo` `` may now incorrectly render to ``<p>\`foo`</p>`` where it should create a proper span in this case.

Reply:
Also be aware of funny case as this one:
```
\``foo``bar`
```

Note the 1st backtick is escaped, but the 2nd still makes beginning of a code span - of length 1 in this case so the closer is after the `bar`, not before it.

Reply:
It produces `<p><code>foo``bar</code></p>` now. So the code span is alright, but there should be a text node with a single backtick preceding it, right?

Reply:
Yes.